client/src/pages/Search/SearchHeader/SearchField.js

```diff
@@ -15,7 +16,7 @@ const classes = {
   searchButton: {
     color: "primary.main",
     margin: 0,
-    height: "fit-content",
+    height: "120%",
```

Comment:
why?

Reply:
The button gets pushed down, I added margintop instead.

Reply:
Try fixing it with align items, don't hard code the height